Add Open edX support doc for enabling waffle flags (#454)
openedx/docs.openedx.org: source/site_ops/how-tos/add-waffle-flag-for-user.rst

How to Enable a Waffle Flag for a User
#######################################

> Follow these steps to enable a waffle flag for internal testing or limited releases:

Reviewer: ```suggestion
Follow these steps to enable a waffle flag for a single user. This may be useful in certain situations, such as internal testing or limited releases:
```

> #. Get the LMS User ID from the Support Tools page. If you do not use the Support Tools Micro-Frontend, find the LMS User ID through other means, such as a database query.

Reviewer: ```suggestion
#. Get the LMS User ID, via a database query or other methods if you have them.
```
